docs/teachertool/checklists/level.json

```
@@ -1,6 +1,19 @@
 {
     "name": "Level",
     "criteria": [
+        {
+            "catalogCriteriaId": "499F3572-E655-4DEE-953B-5F26BF0191D7",
+            "instanceId": "zBovdISYJKVRHRGeI1hiZ",
+            "params": [
+                {
+                    "name": "question",
+                    "value": "Does this illustrate an understanding of how to use conditionals to respond to data from the micro:bit sensors?"
```

This might be semantic and one way might be better for the AI than the other, but I just wanted to ask. Is "using conditionals to respond to data" actually what's going on in this tutorial? I just have a hard time grasping what using conditionals to respond to data looks like since conditions usually don't do the responding. But I'm not sure my suggestion improves it much..
```suggestion
                    "value": "Does this illustrate an understanding of how to use conditionals when responding to data from the micro:bit sensors?"
```